bakerydemo/static/css/font-marcellus.css

```diff
@@ -0,0 +1,8 @@
+@font-face {
+  font-family: 'Marcellus';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Marcellus-Regular'), local('Marcellus Regular'),
+       url('../fonts/Marcellus-Regular.ttf') 
+       
```

Comment:
Could add `font-display: swap` here to show a fallback font whilst the custom font loads https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display

Reply:
Awesome